content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -192,41 +192,41 @@ kubectl delete clusterrolebinding kubeadm:node-autoapprove-bootstrap
 
 After that, `kubeadm join` will block until the admin has manually approved the CSR in flight:
 
+1. Using `kubectl get csr`, you can see that the original CSR is in the Pending state.
```

Comment:
When you make this job a list of several steps, be sure to indent each and every step properly.
In this case, you will need to indent line 195-204 by three spaces, or else line 195 becomes a standalone list with a single item and line 206 starts a different list.

Reply:
Based on your review suggestion, I adjusted to indent line 195-204 by three spaces.
Thank you!